src/Game/UI/Gumps/TopBarGump.cs

```diff
@@ -107,11 +107,22 @@ private TopBarGump() : base(0, 0)
                 new[] { 1, (int) Buttons.GlobalChat }
             };
 
+            cliloc = ClilocLoader.Instance;
+
             string[] texts =
             {
-                ResGumps.Map, ResGumps.Paperdoll, ResGumps.Inventory, ResGumps.Journal, ResGumps.Chat, ResGumps.Help,
-                ResGumps.WorldMap, ResGumps.Info, ResGumps.Debug, ResGumps.NetStats, ResGumps.UOStore,
-                ResGumps.GlobalChat
+                cliloc.GetString(3000430,ResGumps.Map), 
+                cliloc.GetString(3000133,ResGumps.Paperdoll), 
+                cliloc.GetString(3000431,ResGumps.Inventory), 
+                cliloc.GetString(3000129,ResGumps.Journal), 
+                cliloc.GetString(3000131,ResGumps.Chat), 
+                cliloc.GetString(3000134,ResGumps.Help),     
+                cliloc.GetString(1015233,ResGumps.WorldMap), 
+                cliloc.GetString(1079449,ResGumps.Info), 
+                cliloc.GetString(1042237,ResGumps.Debug), 
+                cliloc.GetString(3000169,ResGumps.NetStats), 
```

Comment:
add a whitespace between the clicloc and the comma please

Reply:
> add a whitespace between the clicloc and the comma please

space between cliloc and coma, or comma and ResGumps?